### ISSUE TITLE
Issue finding twilio-sms.conf file

### ISSUE THREAD

**Issue by spauldingportraits** [ericallenpaul/Zabbix-Twilio.sh#1]
trying to implement this in my zabbix install. 

using the script test command works correctly and sends SMS. Once I add it as a media type and try to test it, it fails with an error saying it can't find the configuration file. 

The error I get is as follow's. 
Details Media type test failed.
AccountSid not set, it must be provided in the config file, or on the command line.
Usage: twilio-sms [-v] [-c configfile] [-d callerid] [-u accountsid] [-p authtoken] number [number[number[...]]]

I've provided the -c /usr/lib/zabbix/alertscripts/twilio-sms.conf both in the script name and as a parameter and it still fails to find the script.

**Comment by coconutservices** [comment]
I also had this issue, what is the correct way to setup this script in the media type settings?

**Comment by ericallenpaul** [comment]
This is what mine looks like:

https://github.com/ericallenpaul/Zabbix-Twilio.sh/blob/master/Image%2010.jpg

**Comment by ericallenpaul** [comment]
This is what my action looks like:
https://github.com/ericallenpaul/Zabbix-Twilio.sh/blob/master/Image%2011.jpg

**Comment by ang3lwitch** [comment]
Hello, I did test this out and POC, seems to work great.
my question is, are there any settings via twilio console I will need to set or tweak.

Currently using a free trial account and according to twilio, should be functional with no restrictions.

I do get sms messages to my phone, however it says something along the lines of,

sent via twilio to free account to "my name"

thanks a bunch...other than that this seems fairly easy to implement.
appreciate you.

**Comment by ericallenpaul** [comment]
No. No settings on yeillio need to be tweaked



Sent from my Galaxy



-------- Original message --------

**Comment by ang3lwitch** [comment]
ok, thanks for that
Enjoy the weekend

**Comment by spauldingportraits** [comment]
I know I ended up having to download the latest twillio config file to get
mine to work properly, but everything else worked perfectly once I got that
up and going.

**Comment by ang3lwitch** [comment]
Have a link or location for the new file?

**Comment by spauldingportraits** [comment]
Eric put it in the readme, but it should be here -
https://www.twilio.com/labs/bash/sms

**Comment by ang3lwitch** [comment]
Well, I do get alerts every time I manipulate a trigger.
A message does come to my phone, that is good.

What I am failing to understand or do is to get the whole body of the zabbix alert, i.e. "host is down" or " diskspace is low on {HOST.NAME} etc...

All i get is "Sent from your Twilio trial account: then nothing else. 
Is that by design or am i doing it wrong, lol?

I would not think the trial account would limit my messages, they say it is fully functional.

so my guess is either I am possibly missing an rpm package to assist (CentOS7) and zabbix 5.0.5.

**Comment by spauldingportraits** [comment]
you need to setup your actions accordingly, this is handled in Zabbix with
plenty of documentation on it, the trial account will always send the "
sent from Twilio trial" before the text in the action that gets triggered.

**Comment by ericallenpaul** [comment]
Yep. You just need to make sure you're using the right variables in your action based on the text message you want to see.
My default message is:
{TRIGGER.STATUS} : {TRIGGER.NAME} - {HOSTNAME} ({IPADDRESS})
Trigger severity: {TRIGGER.SEVERITY}

**Comment by frank3427** [comment]
what is the best way to filter our hosts from the SMS, for example I do not want to be altered if dev or testing servers have an event?

**Comment by ericallenpaul** [comment]
Put the hosts in maintenance.

**Comment by vali5k** [comment]
> trying to implement this in my zabbix install.
> 
> using the script test command works correctly and sends SMS. Once I add it as a media type and try to test it, it fails with an error saying it can't find the configuration file.
> 
> The error I get is as follow's. Details Media type test failed. AccountSid not set, it must be provided in the config file, or on the command line. Usage: twilio-sms [-v] [-c configfile] [-d callerid] [-u accountsid] [-p authtoken] number [number[number[...]]]
> 
> I've provided the -c /usr/lib/zabbix/alertscripts/twilio-sms.conf both in the script name and as a parameter and it still fails to find the script.

i got the same problem with my sms alerts. did someone find a solution to this problem?
![image](https://user-images.githubusercontent.com/99742083/154082576-d24d1ccd-6d34-49db-b15f-281ebcd64b5c.png)

**Comment by ericallenpaul** [comment]
Here are some screenshots of my setup. This is Zabbix 3.2 so your screens may be slightly different.
So this is what my action looks like:

***@***.***

And this is my media setup.
***@***.***

The conf file and the script are in the same directory.
Hope this helps.

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows
